docs/decisions/0004-exam-ida.rst

```diff
@@ -0,0 +1,64 @@
+
+.. _Proctoring Services Standard: https://www.imsglobal.org/proctoring-services
+
+.. _Exam Components: https://lucid.app/lucidchart/a4b40637-93f1-47f1-bb17-d68e4ff6f9d9/edit?invitationId=inv_92a6d5c4-d14c-472a-b2bc-64b64d9d45ef
+
+.. _Release Instructions: https://github.com/openedx/edx-proctoring/blob/master/docs/developing.rst#how-do-i-release-edx-proctoring
+
+.. _special-exams-lib: https://github.com/edx/frontend-lib-special-exams 
+
+Decision
+--------
+A new repository and IDA will be created for the exam service. This new exam service will handle all configuration and interaction with third party proctoring tools. It will also be the source of truth for managing a learner's progress through exam setup, completion, and review flows.
+
+* The IDA will function in parallel with the current edx-proctoring library. edx-platform may use either service for a course.
+
+    * Exams should be considered a periphery service. As such, edx-platform should not depend on this service.
+
+* Changes in exam state that would impact other systems such as grades, completion, credit, or certificates should be pushed to those systems via REST endpoints or using events.
+
+* Data about course content or exam configuration will be pushed to the IDA's REST API as part of the studio publish action. We should replicate data instead of reading directly from the CMS or LMS in an ad-hoc manner.
+
+* The IDA will implement a REST API to expose exam and attempt state information to https://github.com/edx/frontend-lib-special-exams.
+
+* The IDA will not include custom APIs or data calls specific to a single proctoring tool.
+
+Consequences
+------------
+
+* The display of learner attempt state in course outlines requires the LMS to directly depend on the proctoring service which is undesireable per the decisions made above. This feature will be simplified to only contain information about the type of exam. Attempt state will be accessible within the section via the frontend-lib-special-exams library.
+
+* The LMS will need a new way to prevent rendering exam content when a learner is not eligible to view those problems that does not rely on the installation of this library.
+
+* Existing proctoring vendors will remain functional using this library but will not by-default be implemented in the new IDA.
+
+* We will need to maintain both proctoring implementations until we can migrate use away from this library and the vendors it specifically supports.
```

Comment:
I thought we were going to migrate everything to the IDA but let the IDA wrap the plugin so that we actually start using the IDA and its interface before we have an actual partner.